packages/cli/src/admin-sdk.ts

```diff
@@ -0,0 +1,224 @@
+import { createClient } from './autogen/client/createClient';
+import fse from 'fs-extra';
+import { kebabCase, omit } from 'lodash';
+import chalk from 'chalk';
+import { readAsJson, getFiles, getDirectories, replaceField } from './utils';
+import cliProgress from 'cli-progress';
+import { createHash } from 'crypto';
+import traverse from 'traverse';
+
+const multibar = new cliProgress.MultiBar(
+  {
+    clearOnComplete: false,
+    hideCursor: true,
+    format: '|{bar}| {name} | {value}/{total}',
+  },
+  cliProgress.Presets.shades_grey
+);
+
+// todo change this to prod
+const root = 'https://qa.builder.io';
```

Comment:
Maybe make this configurable via a flag that gets passed in so when we want to test new features we can deploy to qa and test it

Reply:
I don't think this is possible in current configuration, the client is getting generated at build time using that specific endpoint, the plan is to eventually move all admin-sdk code from here to it's own package, where this could be possible